terminaltables/tables.py

```diff
@@ -92,3 +92,66 @@ class DoubleTable(WindowsTableDouble):
     """Cross-platform table with box-drawing characters. On Windows it's double borders, on Linux/OSX it's unicode."""
 
     pass
+
+
+class GithubFlavoredMarkdownTable(BaseTable):
+    """Github flavored markdown table.
+
+    https://help.github.com/articles/github-flavored-markdown/#tables
+    """
+
+    CHAR_VERTICAL = '|'
+    CHAR_HORIZONTAL = '-'
+
+    def __init__(self, table_data):
```

Comment:
You need to call super(GithubFlavoredMarkdownTable, self).init(table_data) within your __init__().
http://stackoverflow.com/questions/576169/understanding-python-super-with-init-methods